blmath/util/json.py

```diff
@@ -0,0 +1,47 @@
+'''
+If an object defines for_json(), it will be serialized as what that returns.
+    class SerializableThing(object):
+        def for_json(self):
+            return "|SerializableThing|"
+
+'''
+
+from baiji.serialization import json
+from baiji.serialization.json import JSONDecoder
+
+class BlmathJSONDecoder(JSONDecoder):
+    SUPPORTED_TYPES = {'value': 'blmath.value.Value'}
+    def __init__(self):
+        super(BlmathJSONDecoder, self).__init__()
+        self.register(self.decode_supported_types)
+
+    def decode_supported_types(self, dct):
```

Comment:
This is copied over from `core` `bodylabs.serialization.json` and I believe can be deleted from `core` as part of the removal of `measurements` since the only `SUPPORTED_TYPE` is `Value`, which is now part of `blmath`.